src/python_testing/matter_testing_support.py

```diff
@@ -331,6 +332,58 @@ def wait_for_report(self):
             asserts.fail("[AttributeChangeCallback] Attribute {expected_attribute} not found in returned report")
 
 
+def await_sequence_of_reports(report_queue: queue.Queue, endpoint_id: int, attribute: TypedAttributePath, sequence: list[Any], timeout_sec: float):
```

Comment:
Q - why not make this a function of the AttributeChangeAccumulator?

Reply:
We could add a method that calls this. Will do as a follow-up.